### PR TITLE
MGMT-3491 add swagger changes for additional OLM based services - fix

### PR DESCRIPTION
**Pull request by gamli75**

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -3663,12 +3663,9 @@ definitions:
...
       operators:
-        type: array
-        items:
-          $ref: '#/definitions/cluster-operator'
-        x-go-custom-tag: gorm:"-"
-        description: Operators that are associated with this cluster and their properties.
-        x-nullable: true
+          x-go-custom-tag: gorm:"type:text"
+          type: string
+          description: Operators that are associated with this cluster and their properties.
```
Comment (priyanka19-98) [inline swagger.yaml:3668]: why do we need extra spaces here?

**Comment by priyanka19-98** [comment]
@avishayt could you please review the `swagger.yaml` ?

**Comment by avishayt** [comment]
Looks good, just fix spaces

File: swagger.yaml (hunk 1 above)
Comment (gamli75) [reply swagger.yaml]: fixed.

**Review by avishayt — APPROVED** [review]

**Pull request merged**